common/thorhelper/thorrparse.cpp

```diff
@@ -140,6 +142,7 @@ bool isUnicodeMatch(unsigned code, unsigned next)
     case RCCgraph: return u_isprint(next) && !u_isspace(next);
     case RCCpunct: return u_isprint(next) && !(u_isspace(next) || u_isalnum(next));
     case RCCxdigit: return (next < 128) && isxdigit(next);          // should be good enough.
+#endif
     case RCCany:   return true;
     default:
         UNIMPLEMENTED;
```

Comment:
Should probably be changed to 

      throw MakeStringException(99, "System was built without Unicode support");